2025-09-16 17:29:12	SEVERE	ms::open	Exception Reported: Table /scratch/scastro/casa/Tests/reports/wrong.ms does not exist
2025-09-16 17:29:12	SEVERE	ms::open	Exception Reported: Table /scratch/scastro/casa/Tests/reports/wrong.ms does not exist
2025-09-16 17:29:12	SEVERE	setjy::::casa	Task setjy raised an exception of class RuntimeError with the following message: Table /scratch/scastro/casa/Tests/reports/wrong.ms does not exist
